docs/api-reference/cli.md

```diff
@@ -227,6 +228,15 @@ For instance, AWS Lambda requires the user to use `tmp/*` as the scratch
 directory. You can override `quilt3.util.CACHE_PATH`, so that `quilt3 install` will succeed
 in Lambda, by setting the `XDG_CACHE_HOME` environment variable.
 
+```python
+>>> import os
+>>> os.environ["XDG_CACHE_HOME"] = "/foo"
```

Comment:
Also, for a lambda, the best solution is to add `XDG_CACHE_HOME=/tmp/.cache/` to the lambda config, instead of doing it in Python.

Need to actually test it first, though.

Reply:
Fixed - thanks!